proxy/src/main/java/org/dragonet/proxy/network/translator/pc/PCPlayEffectPacketTranslator.java

```diff
@@ -156,5 +192,48 @@
 		breakBlock.put(12, BuiltinSound.BLOCK_SAND_BREAK);
 		breakBlock.put(13, BuiltinSound.BLOCK_GRAVEL_BREAK);
 	}
+	
+	// From Nukkit
+    public static final int PARTICLE_TYPE_BUBBLE = 1;
```

Comment:
Nukkit is disgusting.

Can you make a new enum somewhere when you have time named "ParticleEffect"? Then make the names like "TYPE_CRITICAL" without the PARTICLE prefix.

Maybe in the commons module, somewhere like [here](https://github.com/DragonetMC/DragonProxy/tree/master/commons/src/main/java/org/dragonet/common/data).

Reply:
ok, I do it